include/curl/curl.h

```diff
@@ -94,6 +94,7 @@
    (defined(__FreeBSD_version) && (__FreeBSD_version < 800000)) || \
    (defined(__MidnightBSD_version) && (__MidnightBSD_version < 100000)) || \
     defined(__sun__) || defined(__serenity__)
+    defined(__vxworks__)
```

Comment:
This does not work without adding a trailing `|| \` on the line above, which then makes me suspect you changed this without testing?